.github/workflows/docker-run-tests.yml

```diff
@@ -0,0 +1,34 @@
+name: Run the Tests (unit, api)
+
+on:
+  pull_request:
+    branches:
+      - master
+      - feature/*
+      - fix/*
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        
+      - name: Validate composer.json and composer.lock
+        run: composer validate
```

Comment:
--no-check-publish https://getcomposer.org/doc/03-cli.md#validate 😋

Reply:
It's ugly, even if you don't deploy any artifacts the name and description is required https://getcomposer.org/doc/04-schema.md#name either you modify your test, or you add this information

Reply:
I removed this step as it was defined in the "php" github actions template.